shapes-demo.repos

```diff
@@ -14,4 +14,4 @@ repositories:
     shapes-demo:
         type: git
         url: https://github.com/eProsima/ShapesDemo.git
-        version: v2.3.2
+        version: v2.3.4
```

Comment:
I suppose that the Fast DDS version should be updated as well. But maybe it is better to do it in a new PR.

Reply:
Indeed, better to target this in a new PR

Reply:
I just submitted #39 for that. It'd need to be rebased after this is merged. If we remove it from here (which I support) We need to remember to include there